packages/babel-plugin-transform-inline-consecutive-adds/src/set-collapser.js

```diff
@@ -44,7 +44,7 @@ class SetCollapser extends Collapser {
       if (args.length !== 1) {
         return false;
       }
-      if (checkReference(args)) {
+      if (checkReference(args[0])) {
```

Comment:
It should not expect an array (as Line 44), so I change to `args[0]`  made `Set` works with this PR.